pkg/compiler/compiler_test.go

```diff
@@ -70,12 +70,12 @@ func compileFile(src string) error {
 func readPathAndRunFunc(t *testing.T, f func(string, string)) {
 	infos, err := ioutil.ReadDir(examplePath)
 	if err != nil {
-		t.Fatal(err)
+		require.NoError(t, err)
```

Comment:
You can also omit `if err != nil` here and below.

Reply:
fixed